### ISSUE TITLE
How to plot two CameraDisplays into the same figure with color_bars?

### ISSUE THREAD

**Issue by dneise** [cta-observatory/ctapipe#1192]
When I plot a CamDisplay like this:
```python
d1 = CameraDisplay(camera)
d1.image = dl1.image
d1.add_colorbar()
d1.axes.grid()
```

all ist good:
![image](https://user-images.githubusercontent.com/8200858/69866578-cc50b800-12a4-11ea-88cf-999d96f7a32e.png)

----

But when I want to plot two next to each other like this:
```python
fig, (a1, a2) = plt.subplots(1, 2, figsize=(12, 6))

d1 = CameraDisplay(camera, ax=a1)
d1.image = dl1.image
d1.add_colorbar()
d1.axes.grid()

d2 = CameraDisplay(camera, ax=a2)
d2.image = dl1.image
d2.add_colorbar()
d2.axes.grid()
```

then both colorbars appear in the 2nd axis
![image](https://user-images.githubusercontent.com/8200858/69866669-0de16300-12a5-11ea-9526-c27b1c88ffd4.png)

**Comment by vuillaut** [comment]
Hi @dneise 
You must pass the axes to `colobar`:
```
fig, (a1, a2) = plt.subplots(1, 2, figsize=(12, 6))

d1 = CameraDisplay(camera, ax=a1)
d1.image = dl1.image
d1.add_colorbar(ax=a1)
d1.axes.grid()

d2 = CameraDisplay(camera, ax=a2)
d2.image = dl1.image
d2.add_colorbar(ax=a2)
d2.axes.grid()
```
But since `CameraDisplay` should know its axes, maybe that can be improved...

**Comment by dneise** [comment]
Ah I did not realize it had that argument ... thx

**Comment by MaxNoe** [comment]
Yeah, probably setting ax=self.axes is a saner default in add_colorbar

**Comment by dneise** [comment]
Fixed in #1193

**Comment by dneise** [comment]
Thanks @vuillaut
